doc/source/cookbook.rst

```diff
@@ -56,8 +53,9 @@ These are some neat pandas ``idioms``
 
 .. ipython:: python
 
-   df = pd.DataFrame(
-        {'AAA' : [4,5,6,7], 'BBB' : [10,20,30,40],'CCC' : [100,50,-30,-50]}); df
+   df = pd.DataFrame({'AAA': [4, 5, 6, 7], 'BBB': [10, 20, 30, 40],
+                      'CCC': [100, 50, -30, -50]})
```

Comment:
do you mind moving `BBB` to the next line? I think that will make this more readable